bitnami/nginx/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx
-version: 5.2.4
+version: 5.2.5
```

Comment:
What about bumping the minor version instead? This is adding a new feature, right?

Reply:
Yes, you are correct. I incremented a minor version now, instead of patch increment.